service/api/service_impl.go

```diff
@@ -43,9 +43,17 @@ func NewServiceImpl(cfg config.Config, store persistence.ProcessStore, logger lo
 func (s serviceImpl) StartProcess(
 	ctx context.Context, request xdbapi.ProcessExecutionStartRequest,
 ) (response *xdbapi.ProcessExecutionStartResponse, retErr *ErrorWithStatus) {
+	timeoutUnixSeconds := 10
```

Comment:
I don't think we want to have a default timeout like that. 

Having unlimited timeout is an important feature for uses. If there is no timeout specified, it should be infinite timeout. (same as in Temporal/iWF, Cadence actually require timeout to be set on start API)